[{"name":"SomeException","module":"GHC.Exception","entity":"data"},{"name":"SomeException","module":"GHC.Exception","type":"SomeException","entity":"constructor"},{"name":"Exception","module":"GHC.Exception","entity":"class"},{"name":"toException","module":"GHC.Exception","entity":"method","class":"Exception"},{"name":"fromException","module":"GHC.Exception","entity":"method","class":"Exception"},{"name":"throw","module":"GHC.Exception","entity":"value"},{"name":"ErrorCall","module":"GHC.Exception","entity":"newtype"},{"name":"ErrorCall","module":"GHC.Exception","type":"ErrorCall","entity":"constructor"},{"name":"ArithException","module":"GHC.Exception","entity":"data"},{"name":"Overflow","module":"GHC.Exception","type":"ArithException","entity":"constructor"},{"name":"Underflow","module":"GHC.Exception","type":"ArithException","entity":"constructor"},{"name":"LossOfPrecision","module":"GHC.Exception","type":"ArithException","entity":"constructor"},{"name":"DivideByZero","module":"GHC.Exception","type":"ArithException","entity":"constructor"},{"name":"Denormal","module":"GHC.Exception","type":"ArithException","entity":"constructor"},{"name":"RatioZeroDenominator","module":"GHC.Exception","type":"ArithException","entity":"constructor"}]
